### PR TITLE
feature/cm-772-jenkins-finance-proxy

### PR DESCRIPTION
**Pull request by willwatters**

### CODE REVIEW

File: groups/promo-infrastructure/alb.tf [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+
+resource "aws_lb_target_group_attachment" "epayments-test" {
+  target_group_arn = [for group in module.promo_proxy_alb.target_group_arns : group if can(regex("epayments-test", group))]
+  target_id        = var.epayments_test_server_ip
+  port             = var.epayments_service_port
+
+  depends_on = [
+    module.promo_proxy_alb
+  ]
+}
```
Comment (MartyFox) [inline groups/promo-infrastructure/alb.tf:258]: You have 4 target groups but only 3 attachments, is that correct?

File: groups/promo-infrastructure/profiles/heritage-staging-eu-west-2/vars [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Account details
```
Comment (MartyFox) [inline groups/promo-infrastructure/profiles/heritage-staging-eu-west-2/vars:1]: There should be no heritage profiles in this folder as its going to be Shared Services only.

File: groups/promo-infrastructure/profiles/shared-services-eu-west-2/vars [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+aws_region  = "eu-west-2"
+aws_account = "shared-services"
+
+# shorthand
+account = "shared"
+region  = "euw2"
+
+# Application details
+application = "promo"
+environment = "live"
```
Comment (MartyFox) [inline groups/promo-infrastructure/profiles/shared-services-eu-west-2/vars:11]: should be shared-services

**Review by MartyFox — COMMENTED** [review]
Can you rename the folder from `promo-infrastructure` to `promo-proxy` as we want to use `promo-infrastructure` for the other work

File: groups/promo-infrastructure/alb.tf (hunk 1 above)
Comment (willwatters) [reply groups/promo-infrastructure/alb.tf:258]: Yes - 4 target groups and 3 attachments.  The attachment which searches on epayments_live regex is planned to pick up the target groups tg-promo-epayments-live-fwk-01 and tg-promo-epayments-live-01, which uses the same target IP address.

File: groups/promo-infrastructure/profiles/heritage-staging-eu-west-2/vars (hunk 2 above)
Comment (willwatters) [reply groups/promo-infrastructure/profiles/heritage-staging-eu-west-2/vars:1]: This is now removed as it has it's own group.

File: groups/promo-infrastructure/alb.tf (hunk 1 above)
Comment (willwatters) [reply groups/promo-infrastructure/alb.tf:258]: For example:

<Location /payments-framework/payments-live/notification>
    ProxyPass        http://ch-lfp-pl1.internal.ch:9000/payments-framework/payments/notification/
    ProxyPassReverse http://ch-lfp-pl1.internal.ch:9000/payments-framework/payments/notification/
  </Location>  <Location /payments-live/payments-framework/payments/smartpay/notification/>
    ProxyPass        http://chf-lfp-pl1.internal.ch:9000/payments-framework/payments/smartpay/notification/
    ProxyPassReverse http://chf-lfp-pl1.internal.ch:9000/payments-framework/payments/smartpay/notification/
  </Location>

File: groups/promo-infrastructure/profiles/shared-services-eu-west-2/vars (hunk 3 above)
Comment (willwatters) [reply groups/promo-infrastructure/profiles/shared-services-eu-west-2/vars:11]: As discussed this has been renamed to promo-proxy

**Review by MartyFox — APPROVED** [review]

**Pull request merged**